create.html

```diff
@@ -7,54 +7,75 @@
     <title>Create</title>
     <link rel="stylesheet" href="./styles/index.css" />
   </head>
-  <body class="app-grid h-100vh">
+  <body class="app-grid">
     <h1 class="app-grid__header" id="create-heading">Create</h1>
-    <main class="app-grid__scrollarea m-20">
-      <form action="" aria-labelledby="create-heading">
+    <main class="app-grid__scrollarea">
+      <form class="form" action="" aria-labelledby="create-heading">
         <label for="question-field"
```

Comment:
really nice code and really good to read 👍 just with the label...if you wrap the label around the textarea, then a for is not necessary...but it´s not wrong and even maybe better if you want to change it later ;)

Reply:
Thank you so much! That's a good hint.